### ISSUE TITLE
Internationalization of QGISRed

### ISSUE THREAD

**Issue by wciezak** [qgisred/QGISRed#94]
Hello, 
are you going to internationalize the plugin? Certainly some users would be pleased. I managed to write a python script that can be used to add another language, but only for the shared part of the code (.py and .ui files). Some of the displayed windows come directly from the dll libraries. It would be great if you could compile the dlls in such a way that you can add another language for them.
I'm not a programmer so forgive me if my thoughts are going in the wrong direction.

**Comment by fmartine** [comment]
The internationalization of the QGISRed plugin, taking advantage of the multilingual QGIS platform, is planned to be done when we release version 1.0, which means achieving the first goals proposed at the beginning of this project. We hope to reach those goals during the next year.

For now, we keep the interface and messages in English, and the manual in English and Spanish. We hope to count on your help to translate all the texts into Polish, which I guess is your country of origin.

**Comment by wciezak** [comment]
Dear Proffesor. Of course, I declare my readiness to translate all texts into Polish. As soon as possible, my team and I will do the translation, and we may do so for all future plugin updates. 
Regards, Wojciech Cieżak.

**Comment by fmartine** [comment]
Thank you very much Wojciech for the offer. However, we still have to move the manual to Wikibooks and review its content thoroughly. Once this work is done, it would be time to proceed to translate the texts. You could also translate in future tutorials, web pages, etc. in addition to the interface and messages if you like. But wait until we have the final texts.

By the way, I have visited your profile in RG, and I am very glad to know that you also work at the University. Someday we may also add demand prediction algorithms to the Digital Twin, when the basic procedures to build DT are a little more advanced.

My best regards,

**Comment by wciezak** [comment]
Of course, I'll wait for the final version of the texts. That's right, Professor. I work at the Wrocław University of Technology in the Department of Water Supply and Sewerage. My scientific interests focus mainly on the area of water supply systems (hydraulic models of water supply networks, forecasting the demand for water, research on the flow rate in water pipes). The implementation of water demand forecasting algorithms in hydraulic models of water supply networks is a very good direction. I declare my substantive assistance in this regard. 
Best regards,
